.github/workflows/tests.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         run: $(go env GOPATH)/bin/golangci-lint run
 
       - name: Run unit tests
-        run: go test -v -race -covermode atomic -coverprofile=coverage.txt ./...
+        run: go test -v -race -covermode=atomic -coverprofile=coverage.txt -coverpkg=$(go list ./... | tr "\n" ",") ./...
```

Comment:
This can be simplified to `-coverpkg=./...`

Reply:
I think I tried that variation, but I'll take a look again.